recipes/ipopt/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "Ipopt" %}
+{% set version = "3.12.6" %}
+{% set sha256 = "b1b04fb2aaf424464c00c939fdd0b4075614d8d689b7bbba3cbaedf4a4c997a5" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.coin-or.org/download/pkgsource/{{ name }}/{{ name }}-{{ version }}.tgz
+  sha256: {{ sha256 }}
+  patches:
+    - mumps_mpi.patch
+
+build:
+  skip: True  # [win]
+  number: 0
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - ampl-mp
+    - mumps
+    - metis
+    - scotch
+    - gcc
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+
+  run:
+    - ampl-mp
+    - mumps
+    - metis
+    - scotch
+    - gcc
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+
+test:
+  files:
+    - test
+    - mytoy.nl
+
+about:
+  home: https://projects.coin-or.org/Ipopt/
+  license: EPL
+  license_file: LICENSE
```

Comment:
Seems the license file at this level is just filler.

```
For licensing information for files in this directory, please see the
comments in the header of each file. For licensing of files in each of
the subdirectories, please see the LICENSE file in that subdirectory.
```

Instead would use `license_file: Ipopt/LICENSE`, which appears to be the actual EPL 1.0 license.